acousticbrainz/views/data.py

```diff
@@ -44,21 +46,30 @@ def view_high_level(mbid):
 
 @data_bp.route("/<uuid:mbid>", methods=["GET"])
 def summary(mbid):
-    lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
-
-    info = _get_track_info(mbid, lowlevel['metadata'])
-
-    # Tomahawk player stuff
-    if not ('artist' in lowlevel['metadata']['tags'] and 'title' in lowlevel['metadata']['tags']):
-        tomahawk_url = None
+    try:
+        summary = get_summary_data(mbid)
+    except NotFound:
+        summary = {}
+    
+    info = _get_track_info(mbid, summary['lowlevel']['metadata'] if summary and summary['lowlevel'] else None)
+    if info and summary and summary['lowlevel']:
```

Comment:
I think `and summary['lowlevel']` checks in this line and the line above are redundant.